Formula/opencsg.rb

```diff
@@ -1,5 +1,5 @@
 class Opencsg < Formula
-  desc "The CSG rendering library"
+  desc "Rendering library"
```

Comment:
```suggestion
  desc "CSG rendering library"
```

Reply:
Also already included in the name.

Reply:
If it were up to me I'd include it anyway. It still clarifies the description without making it overly cluttered or unreadable. 

```console
$ brew info opencsg
opencsg: stable 1.4.2 (bottled)
Rendering library
...
```

When I read the above text, I don't immediately see `CSG` anywhere. I think that `CSG rendering library` is a more complete description.

Again, I won't fight hard on this.